avocado/core/app.py

```diff
@@ -38,6 +39,16 @@ def __init__(self):
         # Catch all libc runtime errors to STDERR
         os.environ['LIBC_FATAL_STDERR_'] = '1'
 
+        def sigterm_handler(signum, frame):     # pylint: disable=W0613
+            children = process.get_children_pids(os.getpid())
+            for child in children:
+                try:
+                    process.kill_process_tree(int(child), sig=signal.SIGTERM)
+                except OSError:
```

Comment:
The `kill_process_tree` won't raise OSError, it uses `safe_kill`. There is no need for OSError.